core/block_svg.js

```diff
@@ -238,6 +238,19 @@ Blockly.BlockSvg.prototype.decompose;
  */
 Blockly.BlockSvg.prototype.compose;
 
+/**
+ * An optional boolean for determining whether or not the context menu is
+ * shown on a block.
+ * @type {?boolean}
+ */
+Blockly.BlockSvg.prototype.contextMenu;
```

Comment:
Just curious why we need this here if it is already defined on the block?

Reply:
Good point.